### DISCUSSION TITLE
v0.3

### DISCUSSION THREAD

**Discussion by fabian-lauer** [fabian-lauer/dbus-shelly-3em-smartmeter#30] [Announcements]
After merging some pull-requests (thanks to the supporters) - its just time to release a new version of the script

## What's Changed
* added logging options by @Kotty666 in https://github.com/fabian-lauer/dbus-shelly-3em-smartmeter/pull/13
* Revert "added logging options" by @fabian-lauer in https://github.com/fabian-lauer/dbus-shelly-3em-smartmeter/pull/19
* Fix Logging, 3EM as pvinverter, 3EM Position by @Kotty666 in https://github.com/fabian-lauer/dbus-shelly-3em-smartmeter/pull/22
* Add link to current Eastron powermeter patch by @LeoWinterDE in https://github.com/fabian-lauer/dbus-shelly-3em-smartmeter/pull/26

## New Contributors
* @Kotty666 made their first contribution in https://github.com/fabian-lauer/dbus-shelly-3em-smartmeter/pull/13
* @fabian-lauer made their first contribution in https://github.com/fabian-lauer/dbus-shelly-3em-smartmeter/pull/19
* @LeoWinterDE made their first contribution in https://github.com/fabian-lauer/dbus-shelly-3em-smartmeter/pull/26

**Full Changelog**: https://github.com/fabian-lauer/dbus-shelly-3em-smartmeter/compare/v0.2...v0.3

<hr /><em>This discussion was created from the release <a href='https://github.com/fabian-lauer/dbus-shelly-3em-smartmeter/releases/tag/v0.3'>v0.3</a>.</em>

**Comment by Gorch-de** [comment]
Thanks for your work! The solution is working fine for me.

**Comment by joshua1971** [comment]
Hallo Fabian, habe mein Venus OS neu aufgesetzt und bekomme den Shelly 3EM nicht im Venus angezeigt. Wo kann ich ansetzen?

**Reply by joshua1971** [reply]
2025-03-22 14:44:32,907 root CRITICAL Error in main type Failed to parse: http://admin:xxxxxxx@192.168.178.64/status

User und Passwort sind definitiv richtig, wenn ich mich über den Browser einlogge passt alles???

**Reply by joshua1971** [reply]
Liegt definitiv an user & passwort, ohne geht es problemlos
